README.md

```diff
@@ -1 +1,75 @@
-# Budget_App
\ No newline at end of file
+# Budget App
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget APp ](#-blog-app-)
+  - [🛠 Built With ](#-built-with-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 BUDGET App <a name="about-project"></a>
+
+> "Budget App" is a mobile web application that gives users the power to manage their budget: users can have a list of expenses (e.g bed) associated with a broader category(Home) so that they can know how much they spent on what.
+
+## Built With
+- Ruby
+- Git
+- Github
+- PostgreSQL
+- JavaScript
+- Rubocop
+- RSpec
+- Stylelint
+
+
+## Getting Started
+To get a local copy of the program running, follow these simple steps
+
+
+### Prerequisites
+- IDE, preferably Visual Studio Code
+
+### Install
+- Ruby
+- Rails
+
+### Setup
+- Open a command line terminal (Git bash preferrably) and navigate to a directory where you would like to save the work folder using cd.
+- Clone the repository with git clone https://github.com/shihabrafio/Budget_App
+- To get the curent working tree, git fetch --all, then checkout to "dev" branch.
+- Finally, run the application by doing the following:
+  - navigate to the root directory with a command line preferably ubuntu terminal
+  - run 'bundle install'
+  - run 'rails db:create db:migrate db:seed
+  - run 'rails serve'
+
```

Comment:
To start the Rails app: 

run 'rails server` you meant? Please correct the typo🤓🛠👍